test-framework/junit5-internal/src/main/java/io/quarkus/test/TestUtil.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.UncheckedIOException;
+
+public final class TestUtil {
+
+    private static String version;
+
+    private TestUtil() {
+    }
+
+    /**
+     * Gets the project version from a version.txt (that has been properly setup in maven to contain the version)
+     * This is needed in order to avoid hard-coding Quarkus dependency versions that could break the release process
+     */
```

Comment:
You can also make this class a filtered resource and use `${project.version}` instead

Reply:
I've changed it to use the existing `Version.getVersion()`.